src/JLD.jl

```diff
@@ -870,13 +875,13 @@ JLD.writeas(data::StackFrame) =
 const _where_macrocall = Symbol("@where")
 function expand_where_macro(e::Expr)
     e.head = :where
-    shift!(e.args)
```

Comment:
Did I mess this up in the rebase? That is, is this line supposed to remain and the args are shifted again if macros have sourceloc?